Makefile

```diff
@@ -25,7 +25,7 @@ else
 endif
 
 EXTENSION = pg_shard
-DATA = pg_shard--1.0.sql
+DATA = pg_shard--1.1.sql pg_shard--1.0--1.1.sql
```

Comment:
So how does this work? Will both files be executed on a ``CREATE EXTENSION`` command?

As I understand, and ``ALTER EXTENSION UPDATE`` on a 1.0 installation will add the new UDF.

Should we be moving the UDF to the 1.1 file, and keeping the above line as pg_shard--1.1.sql only? If I run a ``CREATE EXTENSION``, shouldn't I expect the 1.1 file to include everything?
I'm not sure, so I'm just asking.